Signal/src/ViewControllers/ConversationView/ConversationViewItem.h

```diff
@@ -39,12 +39,18 @@ NSString *NSStringForOWSMessageCellType(OWSMessageCellType cellType);
 
 @property (nonatomic, readonly) TSInteraction *interaction;
 
+@property (nonatomic, readonly) BOOL isGroupThread;
+
 @property (nonatomic) BOOL shouldShowDate;
+@property (nonatomic) BOOL shouldHideRecipientStatus;
+
+@property (nonatomic) NSInteger row;
+@property (nonatomic) NSInteger lastRow;
```

Comment:
maybe `previousRow` or `priorRow` to disambiguate from "final row"

Reply:
Done and added a comment.